src/private/php/Assigner.php

```diff
@@ -162,5 +162,34 @@ public static function canUseAssignerSgArray(array $serverGroups): bool {
 
         return false;
     }
+    
+    public static function getCooldownPeriod(): int {
+        return Config::get("assigner_cooldown_seconds");
+    }
+
+    public static function isOnCooldown(): array {
+        $cooldownPeriod = self::getCooldownPeriod();
+        $cldbid = Auth::getCldbid();
+        $cacheKey = "assigner_last_usage_" . $cldbid;
+        $lastUsageTime = CacheManager::i()->get($cacheKey) ?? 0;
+        $currentTime = time();
+        $timeSinceLastUsage = $currentTime - $lastUsageTime;
+
+        if ($timeSinceLastUsage < $cooldownPeriod) {
+            return [
+                "onCooldown" => true,
+                "cooldownRemaining" => $cooldownPeriod - $timeSinceLastUsage
```

Comment:
Suggestion: simply return an int with cooldown remaining. If cooldown > 0, then it's active. Makes it easier to pass and check one value instead of an array of values, but still represents the same data.